src/themes/teams/components/Text/textStyles.ts

```diff
@@ -35,7 +20,6 @@ export default {
       ...(disabled && { color: v.disabledColor }),
```

Comment:
What is the reason that we still have these styles here (for the disabled, error, success etc) that we already have in the base team. Wouldn't it be enough to just change the variables values in teams' theme?

Reply:
Missed this one, fixing it.